cirq/study/trial_result_test.py

```diff
@@ -20,39 +20,38 @@
 import cirq
 
 
-
 def test_repr():
-    v = cirq.TrialResult(params=cirq.ParamResolver({'a': 2}),
-                         measurements={'xy': np.array([[1, 0], [0, 1]])})
-
-    assert repr(v) == ("cirq.TrialResult(params=cirq.ParamResolver({'a': 2}), "
-                       "repetitions=2, measurements={'xy': array([[1, 0],\n"
-                       "       [0, 1]])})")
+    v = cirq.TrialResult.from_single_parameter_set_reps(
+        params=cirq.ParamResolver({'a': 2}),
+        measurements={'xy': np.array([[1, 0], [0, 1]])})
+    cirq.testing.assert_equivalent_repr(v)
```

Comment:
Needs `cirq` and `array` in global_vals.

Reply:
Fixed in a different way.